### ISSUE TITLE
trace function requires the name parameter (it isn't optional), and has an optional expression parameter

### ISSUE THREAD

**Issue by brianpos** [HL7/fhirpath.js#118]
As per the spec, the name parameter is required, and an additional expression option needs to be added
http://hl7.org/fhirpath/#tracename-string-projection-expression-collection

**Comment by plynchnlm** [comment]
I agree the name parameter should be required, and that the expression parameter added in N1 should be supported.

How urgent is this?  Would you like to submit a PR?
